eth/main.cpp

```diff
@@ -79,14 +79,9 @@ void help()
 	cout
 		<< "Usage eth [OPTIONS]" << endl
 		<< "Options:" << endl << endl
-		<< "Operating mode (default is non-interactive node):" << endl;
-#if !ETH_TRUE
-	cout
+		<< "Operating mode (default is non-interactive node):" << endl
 		<< "    import <file>  Import file as a concatenated series of blocks." << endl
-		<< "    export <file>  Export file as a concatenated series of blocks." << endl;
```

Comment:
These were commented out before!
Actually I did not take too close attention to similar things in the other PRs. Could you please re-check?